.github/workflows/ci-cd.yml

```diff
@@ -138,3 +138,48 @@ jobs:
 
   #     - name: Test
   #       run: yarn test:ember-compatibility ${{ matrix.scenario }} --- yarn lint:types --project tsconfig.ts3.json
+
+
+  deploy-documentation:
+    name: Deploy documentation
+    runs-on: ubuntu-latest
+    needs: [lint, test-addon, test-floating-dependencies, try-scenarios]
+    timeout-minutes: 10
+    # Only run on pushes to the main branch or a tag (i.e. ignore pull requests and cron)
+    if: github.event_name == 'push' && (github.ref == 'refs/heads/main' || github.ref.type == 'tag')
+    steps:
+      - name: Check out a copy of the repo
+        uses: actions/checkout@v3
+
+      - name: Set a Git user
+        run: |
+          # Set a Git user for committing
+          git config --global user.name "GitHub Actions"
+          git config --global user.email "actions@users.noreply.github.com"
+
+          # Copy the Git Auth from the local config
+          git config --global "http.https://github.com/.extraheader" \
+            "$(git config --local --get http.https://github.com/.extraheader)"
```

Comment:
To understand how we're deploying the app without setting public and private SSH keys, see https://github.com/ember-learn/ember-octane-vs-classic-cheat-sheet/pull/58.